src/javaRosa.js

```diff
@@ -1980,6 +1973,24 @@ define([
                 widget: iTextIDWidget,
                 widgetValuePath: "constraintMsgItext"
             };
+            databind.constraintMediaIText = function (mugOptions) {
+                return mugOptions.isSpecialGroup ? undefined : {
```

Comment:
Yeah. We also use it in other places (which is where I got this convention from) https://github.com/dimagi/Vellum/blob/97ebfd6860969dc891362dab7f9a1a092408a224/src/javaRosa.js#L2071